src/radical/pilot/agent/radical-pilot-agent-multicore.py

```diff
@@ -5961,6 +5981,11 @@ def sigalarm_handler(signum, frame):
             cfg['lrms_info']['cores_per_node'] = lrms.cores_per_node
             cfg['lrms_info']['agent_nodes']    = lrms.agent_nodes
 
+            # Store some runtime information into the session
+            if 'version_info' in lrms.lm_info:
+                mongo_p.update({"_id": pilot_id},
+                               {"$set": {"lm_info": lrms.lm_info['version_info']}})
+
```

Comment:
Would it make sense to push all of lm_info?  If not, that probably should be renamed to lm_version or something.

This is nitpicking.  The PR looks good to me in general!